k8s/namespaces/unspec/unspec-service.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: unspec-service
+  namespace: unspec
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: unspec-service
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/unspec-service
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/unspec/service:prod-d6cd6abc
+      aadIdentityName: unspec
```

Comment:
It is already in https://github.com/hmcts/hmcts-charts/blob/master/stable/unspec-service/values.yaml#L17-L18
```suggestion
    
```